stem_cell/tests/test_differentiator.py

```diff
@@ -39,6 +49,12 @@ def get_version_control_token(self):
     def set_version_control_token(self, token):
         self.version_control_token = token
 
+    def set_version_control_private_key(self, private_key):
```

Comment:
1 min of googling reveals this kind of stuff to get behaviours onto get/set pattern

```python
class MyClass(object):

    def __init__(self):
        self._my_attribute = 0

    @property
    def my_attribute(self):
        # Do something if you want
        return self._my_attribute

    @my_attribute.setter
    def my_attribute(self, value):
        # Do something if you want
        self._my_attribute = value
```